en/user_guide/node_configuration.rst

```diff
@@ -0,0 +1,122 @@
+Node configuration
+==================
+
+Here are the available options in the node configuration. Some of them are required
+in order to build a node and must therefore be provided.
```

Comment:
I think it would be a bit heavy in this context. What do you say about adding a link to the node page ?